incubator/cassandra/README.md

```diff
@@ -1,60 +1,84 @@
+Cassandra is a distributed NoSQL database management system designed to handle
+large amounts of data across many commodity servers, providing high
+availability with no single point of failure.
+
+## Installing Chart
+ 
+Installation of this Chart requires PV (Persistent Volume) storage.  You may
+need to create a storage class before you install.  To create this, see the
+[Persisting Data](#persisting-data) section.
 
 ```bash
 helm install --namespace "cassandra" -n "cassandra" incubator/cassandra
 ```
 
-After installation succuess, you can get a status of Chart
+Once installed, you can get the status of the Chart with:
 
 ```bash
 helm status "cassandra"
 ```
 
-If you want to delete your Chart, use this command
+To uninstall this Chart:
+
 ```bash
 helm delete  --purge "cassandra"
 ```
 
-## Persist data
-You need to create `StorageClass` before able to persist data in persistent volume. 
-To create a `StorageClass` on Google Cloud, run the following
+## Persisting Data
+
+In order to persist data you need to create a `StorageClass`.  This allows the chart
+to create the required storage automatically on your behalf.
+
+On GCE (Google Container Engine):
 
 ```bash
 kubectl create -f sample/create-storage-gce.yaml
 ```
 
-And set the following values in `values.yaml`
+Then set the following values in `values.yaml`
 
 ```yaml
 persistence:
   enabled: true
 ```
 
-If you want to create a `StorageClass` on other platform, please see documentation here [https://kubernetes.io/docs/user-guide/persistent-volumes/](https://kubernetes.io/docs/user-guide/persistent-volumes/)
+If you want to create a `StorageClass` on other platform, please see
+documentation here
+[https://kubernetes.io/docs/user-guide/persistent-volumes/](https://kubernetes.io/docs/user-guide/persistent-volumes/)
+
 
+## Sizing Cassandra Cluster
 
-## Install Chart with specific cluster size
-By default, this Chart will create a cassandra with 3 nodes. If you want to change the cluster size during installation, you can use `--set config.cluster_size={value}` argument. Or edit `values.yaml`
+This Chart defaults to creating a Cassandra cluster with 3 nodes.  To increase
+the cluster size during installation you can use `--set
+config.cluster_size={value}`, or edit `values.yaml` and set it there.
 
-For example:
-Set cluster size to 5
+Example (Set cluster size to 5):
 
 ```bash
 helm install --namespace "cassandra" -n "cassandra" --set config.cluster_size=5 incubator/cassandra/
 ```
 
-## Install Chart with specific resource size
-By default, this Chart will create a cassandra with CPU 2 vCPU and 4Gi of memery which is suitable for development environment.
-If you want to use this Chart for production, I would recommend to update the CPU to 4 vCPU and 16Gi. Also increase size of `max_heap_size` and `heap_new_size`.
-To update the settings, edit `values.yaml`
+## Modifying resource limits
```

Comment:
Same here i would either go with the older heading 
`## Install Chart with specific resource size`
or say 
`Modifying the default resource limits`